src/migrations/1741025559300_scene-admin.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate'
+
+export const SceneAdminColumns: ColumnDefinitions = {
+  id: { type: 'uuid', primaryKey: true },
+  place_id: { type: 'text', notNull: true },
+  admin: { type: 'text', notNull: true },
+  added_by: { type: 'text', notNull: true },
+  created_at: { type: 'bigint', notNull: true },
+  active: { type: 'boolean', notNull: true, default: true }
+}
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.createTable('scene_admin', SceneAdminColumns)
+  pgm.createIndex('scene_admin', ['place_id', 'admin'], {
```

Comment:
the unique index I was asking before is already created, we should remove the check that validates if the relation already exists